clients/spark/core/src/test/resources/parser-test/generate-sst.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Generate sst files that will serve as an input jpebble parser tests
+export CGO_ENABLED=0
+cd /local
+go run sst_files_generator.go
+echo "done"
```

Comment:
This line means that the shell script always succeeds, even when it fails.  Please use e.g. `#!/bin/sh -e`, and maybe even `#!/bin/sh -eu`.  Run `help set` in `bash` for more about some useful shell options (you can also say `set -e; set -u` and similar to set these once the shell is already running.